docs/src/pages/overview.md

```diff
@@ -3,4 +3,24 @@ id: overview
 title: Overview
 ---
 
-Monika is an open source and free synthetic monitoring command line application. The name Monika stands for "**Moni**toring Ber**ka**la", which means "periodic monitoring" in the Indonesian language.
+Monika is an open source and free synthetic monitoring command line application. The name Monika stands for "**Moni**toring Ber**ka**la", or "periodic monitoring" in the Indonesian language.
+
+## Motivation
+
+It goes without saying, whether a front facing website or a backend service, everyone would like their services to be fast, performant and reliable. Good design, engineeering excellence and proper processes will contribute to these goals. Not to be overlooked is the importance tooling has on project succeses. Sadly good tools are not always budgeted, often relegated to something "nice to have" rather than "essential to have". This is our motivation. We believe the proper tools should go hand in hand with great teams and engineering processes. We believe tooling should be integrated with the development process and extends after deployment.
+
+With this in mind Monika was created, a sythetic monitoring tool.
+
+## Not just another tool
+
+There are plenty of free monitoring tools online, but many fall short our requirements. Free uptime monitors exists, but these monitors only ping for service availability. Users don't typically use services with pings. There are also plenty of real time monitoring tools. These tools by their very nature require real users which make them less suitable during development. With Monika, you can instead synthesize load scenarios during development, then use the same scenarios after deployment. Synthetic monitors allow QA teams for instance, to generate complex user flows for qualification. The same flows can be used to ensure production performance, and all without the need to install agents or third party libraries.
+
+In addition, Monika will be easy to deploy. You can deploy Monika to specific regions and receive notifications when the quality of service degrades. For free.
```

Comment:
You can deploy Monika to your own servers in different regions and receive notification when the quality of service from any of those regions degrades.